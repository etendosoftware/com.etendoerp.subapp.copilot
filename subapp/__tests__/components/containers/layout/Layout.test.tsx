import React from 'react';
import { Text } from 'react-native';
import renderer from 'react-test-renderer';

import Layout from '../../../../src/components/containers/layout/Layout.tsx';

describe('Running Test for Layout', () => {
  it('Check Layout', () => {
    const tree = renderer
      .create(
        <Layout
          style={{ backgroundColor: 'red' }}
          isKeyboadAvoid
          isScrollVertical>
          <Text>Test</Text>
        </Layout>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
