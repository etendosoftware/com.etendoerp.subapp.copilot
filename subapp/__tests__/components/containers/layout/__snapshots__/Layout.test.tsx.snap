// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Running Test for Layout Check Layout 1`] = `
<View
  onLayout={[Function]}
  style={
    [
      {
        "flex": 1,
      },
      {
        "paddingBottom": 0,
      },
    ]
  }
>
  <RCTScrollView>
    <View>
      <View
        style={
          [
            {
              "alignItems": "center",
              "display": "flex",
              "flex": 1,
              "height": "100%",
              "justifyContent": "flex-start",
              "paddingHorizontal": 32,
              "paddingVertical": 28,
              "width": "100%",
            },
            {
              "backgroundColor": "red",
            },
          ]
        }
      >
        <Text>
          Test
        </Text>
      </View>
    </View>
  </RCTScrollView>
</View>
`;
